Makefile

```diff
@@ -207,8 +208,16 @@ setup-new-sdk:
 .PHONY: build-client-java
 build-client-java:
 	make build-client sdk_language=java tmpdir=${TMP_DIR}
-	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c 'gradle fmt'"
+	make run-in-docker sdk_language=java image=busybox:${BUSYBOX_DOCKER_TAG} command="/bin/sh -c 'chmod +x ./gradlew'"
+	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c './gradlew fmt'"
+
+.PHONY: test-client-java
+test-integration-client-java: build-client-java
+	docker container rm --force openfga-for-java-client || true
+	docker run --detach --name openfga-for-java-client -p 8080:8080 openfga/openfga run
+	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c './gradlew test-integration'"
+	docker container rm --force openfga-for-java-client
 
 .PHONY: test-client-java
 test-client-java: build-client-java
-	# ... any custom test code ...
+	make run-in-docker sdk_language=java image=gradle:${GRADLE_DOCKER_TAG} command="/bin/sh -c './gradlew build'"
```

Comment:
```suggestion
```

You can remove this line for now, when we do add tests we can add it.

Without this line, it will just build, so this is just repeating the build step